epc-data-import.md

```diff
@@ -0,0 +1,45 @@
+# Challenge: EPC Data Import
+
+## Getting Started
+
+- Register a user at [epc.opendatacommunities.org](https://epc.opendatacommunities.org/#register) so that you can access the EPC data
+- This challenge is concerned with the "Domestic EPC". There is a downloadable zip (~3Gb+) for around 18,000,000 EPC records.
+
+## Problem
+
+We'd like to import the data as part of a regular, repeatable, automated data pipeline.
+
+- Within the zip, we want to import data from all the `certificates.csv` files
+- Within each `certificates.csv` we only want the columns `LMK_KEY`, `LODGEMENT_DATE`, `TRANSACTION_TYPE`, `TOTAL_FLOOR_AREA`, `ADDRESS`, `POSTCODE`, all other columns can be ignored.
+- We'd like to write the results to a database (can be any flavour of database you prefer)
+
+## Scope
+
+- We have AWS resources and a Kubernetes cluster (including an ELK stack and Prometheus) at our disposal (can be included as part of your decisions and solutions)
+- The data changes/updates every quarter and may increase in size
+- For each EPC entry we have to call an external API with the `ADDRESS` column to get a lat/lng. This API isn't provided nor do you need to code for it, but do be aware that the response time is 250ms for a single request when designing your solution
```

Comment:
Should we allow them to assume geocoding requests could be batched? Or should we let them ask the question?

Reply:
I did wonder that. I left it open but had considered saying that the service can take some amount of load.